modules/apps/content-dashboard/content-dashboard-web/src/main/java/com/liferay/content/dashboard/web/internal/item/FileEntryContentDashboardItem.java

```diff
@@ -107,6 +107,26 @@ public FileEntryContentDashboardItem(
 		_portal = portal;
 	}
 
+	@Override
+	public List<Version> getAllVersions(Locale locale, int status) {
+		Stream<FileVersion> stream = _fileEntry.getFileVersions(
```

Comment:
Hey @jkappler !!

What do you think about limit the versions results? It is possible to break the frontend if we return too many versions?

Thanks!!

Reply:
Will be handled in a new story